src/Views/FirmwareView.vala

```diff
@@ -120,12 +120,12 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
                 add_device (devices[i]);
             }
 
-            placeholder_alert_view.title = _("Firmware Updates Are Not Available");
```

Comment:
Sentence in title casing. Weird sentence at that.
Are they anything but unavailable?